examples/hashmap.rs

```diff
@@ -266,15 +265,15 @@ impl state_machine::StateMachine for HashmapStateMachine {
         // Handle
         let response = match message {
             Get(key) => {
-                let old_value = &self.map.get(&key).map(|v| v.clone());
+                let old_value = &self.map.get(&key).cloned();
                 serde_json::to_string(old_value)
             },
             Put(key, value) => {
                 let old_value = &self.map.insert(key, value);
                 serde_json::to_string(old_value)
             },
             Cas(key, old_check, new) => {
-                if *self.map.get(&key).unwrap() == old_check {
+                if self.map[&key] == old_check {
```

Comment:
I'm actually against this change. The whole `.get(&key).unwrap()` should be refactored so it doesn't crash. Using `self.map[&key]` means if the key doesn't exist we implicitly crash. At least in the old case we are more explicit about the crash.